3rdparty/bazel-rules-curl/repos.bzl

```diff
@@ -1,4 +1,4 @@
-"""Adds repostories/archives."""
+"""Adds repositories/archives."""
 
 ########################################################################
 # DO NOT EDIT THIS FILE unless you are inside the
```

Comment:
Can I ask you to propagate these changes back to the original repo(s) if you haven't already so we don't end up having to fix this again? :)

Reply:
This goes for the others `repos.bzl` files too.

Reply:
> 

Sure!